src/olympia/abuse/views.py

```diff
@@ -191,59 +261,17 @@ def cinder_webhook(request):
     set_user(UserProfile.objects.get(pk=settings.TASK_USER_ID))
 
     try:
-        if request.data.get('event') != 'decision.created':
-            log.info('Non-decision payload received: %s', str(request.data)[:255])
-            raise ValidationError('Not a decision')
-
         if not (payload := request.data.get('payload', {})):
```

Comment:
Could you add a check if payload is not a dict ? I got a 500 locally because I was calling the webhook with multipart data instead of json, and somehow that caused `payload` to be present, but as a string.... It wasn't happening before because the `request.data.get('event') != 'decision.created':` condition was causing us to skip the entire thing early.

Reply:
https://github.com/mozilla/addons-server/pull/22795/commits/61f4c6747a818a78e0affbc4b37f87a40e9b7f7f